src/stim/io/measure_record_reader.h

```diff
@@ -90,7 +113,7 @@ struct MeasureRecordReaderFormatB8 : MeasureRecordReader {
 
     MeasureRecordReaderFormatB8(FILE *in, size_t bits_per_record);
 
-    size_t read_bytes(PointerRange<uint8_t> data) override;
+    size_t read_bits_into_bytes(PointerRange<uint8_t> out_buffer) override;
```

Comment:
The reason I changed the name is because the old name made it ambiguous that the bytes argument is being written into, instead of read out of. It also was counting bits but had bytes in the name, so the result appeared to be off by a factor of 8. I would change the writer API to be more concise rather than restore the old name.

Reply:
The first point is addressed by changing argument name (that I agree with). The second one (return value) is indeed not great. Changing writer API to match SGTM.